packages/react/src/css-map/index.ts

```diff
@@ -0,0 +1,8 @@
+import type { CSSProps, CssObject } from '../types';
+import { createSetupError } from '../utils/error';
+
+export default function cssMap<T extends string, TProps = unknown>(
```

Comment:
You can probably handle this when you make the types stricter but a TSDoc here would be great (like your existing one for `evaluateCSSMap`)

Reply:
Good idea.

Reply:
I encounter an issue with Flowgen when trying to make the types stricter. [Flowgen crushes](https://github.com/joarwilk/flowgen/issues/188) when it converts template literal types.
The only thing (super hacky) I can think of is to skip this file during flowgen. And then I manually create cssmap.js.flow. 
I am going to leave as-is, and revisit it later if it's okay with you?